python-modules/cis_change_service/cis_change_service/api.py

```diff
@@ -0,0 +1,83 @@
+import logging
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flask_cors import CORS
+from flask_cors import cross_origin
+from cis_change_service import get_config
+from cis_change_service import profile
+from cis_change_service.idp import requires_auth
+from cis_change_service.idp import AuthError
+from cis_change_service import __version__
+
+from cis_publisher import operation
+
+
+app = Flask(__name__)
+config = get_config()
+logger = logging.getLogger(__name__)
+
+CORS(
+    app,
+    allow_headers=(
+        'x-requested-with',
+        'content-type',
+        'accept',
+        'origin',
+        'authorization',
+        'x-csrftoken',
+        'withcredentials',
+        'cache-control',
+        'cookie',
+        'session-id',
+    ),
+    supports_credentials=True
+)
+
+
+@app.errorhandler(AuthError)
+def handle_auth_error(ex):
+    response = jsonify(ex.error)
+    response.status_code = ex.status_code
+    return response
+
+
+@app.route('/')
+def index():
+    return 'Mozilla Change Integration Service Endpoint'
```

Comment:
not 403?

Reply:
For / it's kinda nice to have something that returns the version and status code 200 so we can health check it.

Reply:
good point